### PR TITLE
docs(observability): create a token

### PR DESCRIPTION
**Pull request by nedaehm**
### Your checklist for this pull request

- [ ] Check that the commit messages match our requested structure.
- [ ] Name your pull request according to the [contribution guidelines](../docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: observability/cockpit/how-to/create-token.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+This page shows you how to create [tokens](/observability/cockpit/concepts/#tokens), to grant other cloud providers’ services and platforms access to your [Cockpit](/observability/cockpit/concepts/#cockpit).
+
+<Message type="requirement">
+
+ - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+ - You have activated your [Cockpit](/observability/cockpit/how-to/activate-cockpit)
+
+</Message>
+
+1. Log into your [Scaleway Console](https://console.scaleway.com/) account.
```
Comment (bene2k1) [inline observability/cockpit/how-to/create-token.mdx:24]:
```suggestion
1. Log into your [Scaleway console](https://console.scaleway.com/) account.
```

File: observability/cockpit/how-to/create-token.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+3. Click the **Tokens** tab.
+
+4. Click **Generate new token** to create a token. A pop-up displays prompting you to create a token.
+
+5. Choose a name for your token or use the already-generated name.
+
+6. Select the permissions you wish to apply to your token.
+
+7. Click **Generate token** to generate your token. A pop-up displays with your secret key. Make sure to save your secret key before closing the pop-up.
```
Comment (bene2k1) [inline observability/cockpit/how-to/create-token.mdx:36]: maybe add the "make sure..." in a message box here?

File: observability/cockpit/how-to/create-token.mdx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+meta: How to create a token
+  title: How to create a token
+  description: This page shows you how to create a token
+content:
+  h1: How to to create a token
```
Comment (nedaehm) [inline observability/cockpit/how-to/create-token.mdx:6]:
```suggestion
  h1: How to create a token
```

File: observability/cockpit/how-to/create-token.mdx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+2. Click **Cockpit** in the side menu. The **Cockpit** overview page displays.
+
+3. Click the **Tokens** tab.
+
+4. Click **Generate new token** to create a token. A pop-up displays prompting you to create a token.
+
+5. Choose a name for your token or use the already-generated name.
+
+6. Select the permissions you wish to apply to your token.
```
Comment (nedaehm) [inline observability/cockpit/how-to/create-token.mdx:34]: Add a message type important to warn the user they must select at least one permission

File: observability/cockpit/how-to/create-token.mdx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+7. Click **Generate token** to generate your token. A pop-up displays with your secret key.
+
+<Message type="important">
+
+ Make sure to save your secret key before closing the pop-up.
+
+</Message>
+
+You newly-created token displays in the list of your tokens.
```
Comment (RoRoJ) [inline observability/cockpit/how-to/create-token.mdx:50]:
```suggestion
Your newly-created token displays in the list of your tokens.
```

File: observability/cockpit/how-to/create-token.mdx [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+This page shows you how to create [tokens](/observability/cockpit/concepts/#tokens), to grant other cloud providers’ services and platforms access to your [Cockpit](/observability/cockpit/concepts/#cockpit).
+
+<Message type="requirement">
+
+ - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+ - You have activated your [Cockpit](/observability/cockpit/how-to/activate-cockpit)
+
+</Message>
+
+1. Log into your [Scaleway console](https://console.scaleway.com/) account.
```
Comment (nedaehm) [inline observability/cockpit/how-to/create-token.mdx:24]:
```suggestion
```

File: observability/cockpit/how-to/create-token.mdx [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+2. Click **Cockpit** in the side menu. The **Cockpit** overview page displays.
```
Comment (nedaehm) [inline observability/cockpit/how-to/create-token.mdx:26]:
```suggestion
1. Click Cockpit in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```

File: observability/cockpit/how-to/create-token.mdx [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+3. Click the **Tokens** tab.
```
Comment (nedaehm) [inline observability/cockpit/how-to/create-token.mdx:28]:
```suggestion
2. Click the **Tokens** tab.
```

File: observability/cockpit/how-to/create-token.mdx [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+4. Click **Generate new token** to create a token. A pop-up displays prompting you to create a token.
```
Comment (nedaehm) [inline observability/cockpit/how-to/create-token.mdx:30]:
```suggestion
3. Click **Generate new token** to create a token. A pop-up displays prompting you to create a token.
```

File: observability/cockpit/how-to/create-token.mdx [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+5. Choose a name for your token or use the already-generated name.
```
Comment (nedaehm) [inline observability/cockpit/how-to/create-token.mdx:32]:
```suggestion
4. Choose a name for your token or use the already-generated name.
```

File: observability/cockpit/how-to/create-token.mdx [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+
+6. Select the permissions you wish to apply to your token.
```
Comment (nedaehm) [inline observability/cockpit/how-to/create-token.mdx:34]:
```suggestion
5. Select the permissions you wish to apply to your token.
```

File: observability/cockpit/how-to/create-token.mdx [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+
+  <Message type="important">
+
+   You must select at least one permission.
+
+  </Message>
+
+7. Click **Generate token** to generate your token. A pop-up displays with your secret key.
```
Comment (nedaehm) [inline observability/cockpit/how-to/create-token.mdx:42]:
```suggestion
6. Click **Generate token** to generate your token. A pop-up displays with your secret key.
```

**Review by RoRoJ — APPROVED** [review]

File: observability/cockpit/how-to/create-token.mdx [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+
+<Message type="requirement">
+
+ - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+ - You have activated your [Cockpit](/observability/cockpit/how-to/activate-cockpit)
+
+</Message>
+
+
+1. Click Cockpit in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```
Comment (RoRoJ) [inline observability/cockpit/how-to/create-token.mdx:25]:
```suggestion
1. Click **Cockpit** in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```

File: observability/cockpit/how-to/create-token.mdx [hunk 14]
```diff
@@ -0,0 +1,54 @@
+---
+meta: How to create a token
```
Comment (ldecarvalho-doc) [inline observability/cockpit/how-to/create-token.mdx:2]: ça s'appelle juste token? comme on a d'autres types de token chez scw ça pourrait être bien de spécifier quel type c'est celui-la

**Pull request merged**